### ISSUE TITLE
It is recommended to upgrade the TeamTalk5PC client to Qt 5.12 LTS

### ISSUE THREAD

**Issue by dingpengyu** [BearWare/TeamTalk5#406]
hello bjoerndrasmussen
I just read the Qt 5.12 LTS release article, in which I saw an update on accessibility, the entries are as follows: Windows, we are now supporting Windows UI Automation, allowing Qt-based UWP applications to operate with accessibility and programmatic UI control Tools. Does the official have plans to upgrade qt5.12 lts?
[Qt 5.12 LTS Released](https://blog.qt.io/blog/2018/12/06/qt-5-12-lts-released/)

**Comment by dingpengyu** [comment]
cc
@beqabeqa473

**Comment by bear101** [comment]
TeamTalk 5.4 uses Qt 5.12

**Comment by dingpengyu** [comment]
> TeamTalk 5.4 uses Qt 5.12
thank

**Comment by bear101** [comment]
Hi @dingpengyu , if you try the latest TeamTalk v5.4 beta then it has been updated to Qt 5.13. Do you see any improvement with regard to accessibility: http://bearware.dk/beta/TeamTalk_v5.4.0.4961_Setup.exe

**Comment by beqabeqa473** [comment]
hi bjoern.

i can mostly see improvements in terms of accessibility as uia is now
used in qt 5.12+.

i have one suggestion/ask.
could you please change control which is ment to select servers and
usernames in connection dialog to another select like editable
combobox or so.
because this type of element is not accessible for screenreaders.

thank you.

On 5/18/19, Bjørn Damstedt Rasmussen <notifications@github.com> wrote:
> Hi @dingpengyu , if you try the latest TeamTalk v5.4 beta then it has been
> updated to Qt 5.13. Do you see any improvement with regard to accessibility:
> http://bearware.dk/beta/TeamTalk_v5.4.0.4961_Setup.exe
>
> --
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub:
> https://github.com/BearWare/TeamTalk5/issues/406#issuecomment-493647779


-- 
with best regards beqa
